### PR TITLE
cilium-tetragon

### PR DESCRIPTION
**Pull request by resmaa**
**Pack Cilium Tetragon**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1005#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by roshanirathi — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1005#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1005#pullrequestreview-1197268945" title="Approved">roshanirathi</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-cilium-tetragon.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cilium-tetragon.md:19]:
```suggestion
[Tetrogon](https://github.com/cilium/tetragon) is an eBPF based security observability and runtime enforcement tool. eBPF is used to safely and efficiently extend the kernel's capabilities without requiring changing kernel source code or load kernel modules. Tetragon is a Cilium community open-source project that enables profound visibility with filtering and aggregation with the eBPF collector support to deliver visibility at depth with minimal overhead.
```

Added link to Tetragon

File: content/docs/06-integrations/00-cilium-tetragon.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cilium-tetragon.md:21]:
```suggestion
Palette supports the Cilium Tetragon as an add-on pack for monitoring services. Refer to the [create cluster profile](/cluster-profiles/task-define-profile#overview) page for more information on how to use an add-on pack.
```
Language cleanup

**Review by karl-cardenas-coding — APPROVED** [review]
Minor changes suggested but otherwise it looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1005#pullrequestreview-1197711449" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1005#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1005#pullrequestreview-1197268945" title="Approved">roshanirathi</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/06-integrations/00-cilium-tetragon.md
Comment (hausmckinney) [reply content/docs/06-integrations/00-cilium-tetragon.md:21]: Remove "the". 

Palette supports Cilium Tetragon as an add-on pack for monitoring services. Refer to the [create cluster profile](/cluster-profiles/task-define-profile#overview) page for more information on how to use an add-on pack.

**Pull request merged**
